lib/chef/mixin/windows_architecture_helper.rb

```diff
@@ -42,6 +42,24 @@ def wow64_architecture_override_required?(node, desired_architecture)
           is_i386_process_on_x86_64_windows?
       end
 
+      def forced_32bit_override_required?(node, desired_architecture)
```

Comment:
A functional spec for this one would be good, one that covered the 32-bit Ruby case and the 64-bit Ruby case, since this function would return true in one case and false in the other.

A unit test would also be good -- it might fit along with existing unit specs for this clasee.